### ISSUE TITLE
reading github.com/pion/mediadevices/go.mod at revision v0.0.0: unknown revision v0.0.0

### ISSUE THREAD

**Issue by ZeoWorks** [wawesomeNOGUI/RemoteDesktopForGaming#3]
Hi, upon attempting to build the .go source, it requests;
`screenGrabWebRTC.go:23:2: no required module provides package github.com/pion/mediadevices/examples/internal/signal; to add it:
	go get github.com/pion/mediadevices/examples/internal/signal
`

Upon running;
go get github.com/pion/mediadevices/examples/internal/signal

This error occurs;
`go: github.com/pion/mediadevices/examples@v0.0.0-20220824115655-3bec69bbf884 requires
	github.com/pion/mediadevices@v0.0.0: reading github.com/pion/mediadevices/go.mod at revision v0.0.0: unknown revision v0.0.0
`

**Comment by wawesomeNOGUI** [comment]
Hi! Sorry I left this repository in a kind of bad state for compiling the code yourself. I just updated it so if you clone the repository you should be able to compile it. Let me know if it works!

**Comment by ZeoWorks** [comment]
Hi, it now builds and runs correctly but unfortunately there is no video stream upon connecting. :)

**Comment by wawesomeNOGUI** [comment]
Hi! Yeah I need to check on that error, try scrolling the page and see if the stream starts playing

**Comment by ZeoWorks** [comment]
Thank you, yes I've tried this however the same issue occurs sir. :)

**Comment by wawesomeNOGUI** [comment]
Did you change this line in index.html to your IP? https://github.com/wawesomeNOGUI/RemoteDesktopForGaming/blob/54963b441d76912b0821172ad79effcc8efa8625/public/index.html#L128

**Comment by ZeoWorks** [comment]
Yes sir, Even occurs on the same machine.
I'd like to clarify that the previous prebuilt exe had no issues.

**Comment by wawesomeNOGUI** [comment]
Oh that's good the old exe still works. Is there any error output from the go console or browser console?

**Comment by ZeoWorks** [comment]
No, none at all.  It connects absolutely fine.  I think the issue is within the prebuilt x264?

**Comment by wawesomeNOGUI** [comment]
What system are you running the program on? I built x264 for Windows 64bit

**Comment by ZeoWorks** [comment]
Windows 10 x64 like yourself

**Comment by ZeoWorks** [comment]
Does downloading this project fresh and compiling work for you?

**Comment by wawesomeNOGUI** [comment]
It does, I was setting up a new laptop so I made sure I could clone this repo and compile it there.

**Comment by wawesomeNOGUI** [comment]
Oh what go/ gcc version are you using?

**Comment by wawesomeNOGUI** [comment]
Ohhh I am getting an error on my laptop that says GetDIBits failed, I'll look into that

**Comment by ZeoWorks** [comment]
Yes sir, getting the same error.
`2022/09/05 10:30:06 Track (ID: 13503cd6-c517-49a5-8dd0-859f4caae8e4) ended with error: GetDIBits failed`

**Comment by ZeoWorks** [comment]
Just to clarify; if I take out the windows input code and try building on Linux, it works with libx264-dev.
Could it be that the issue is appearing due to your build of x264 specifically on Windows?

**Comment by wawesomeNOGUI** [comment]
Hi! I fixed the error and running on windows works now. For some reason, the screen size was always being passed as 0,0. My fix was to edit github.com/kbinani/screenshot with a new screenshot_windows.go I'll keep looking for a better fix, but that seems to work for now
